docs/data-ai/train/capture-images.md

```diff
@@ -0,0 +1,283 @@
+  host: MACHINE_ADDRESS,
+  credential: {
+    type: "api-key",
+    payload: API_KEY,
+  },
+  authEntity: API_KEY_ID,
+});
+
+const client: ViamClient = await createViamClient({
+  credential: {
+    type: "api-key",
+    payload: API_KEY,
+  },
+  authEntity: API_KEY_ID,
+});
+
+const dataClient = client.dataClient;
+
+const camera = new Viam.CameraClient(machine, CAMERA_NAME);
+
+// Capture image
+const imageFrame = await camera.getImage();
+
+// Upload binary data
+const now = new Date();
+const fileId = await dataClient.binaryDataCaptureUpload({
+  partId: PART_ID,
+  componentType: "camera",
+  componentName: CAMERA_NAME,
+  methodName: "GetImages",
+  dataRequestTimes: [now, now],
+  fileExtension: ".jpg",
+  binaryData: imageFrame,
+});
+
+// Cleanup
+await machine.disconnect();
+dataClient.close();
+```
+
+{{% /tab %}}
+{{% tab name="Flutter" %}}
+
+To capture an image and add it to your **DATA** page, fetch an image from your camera through your machine.
+Pass that image and an appropriate set of metadata to [`dataClient.binaryDataCaptureUpload`](/dev/reference/apis/data-client/#binarydatacaptureupload):
+
+```dart
+const String CAMERA_NAME = '<camera-name>';
+const String MACHINE_ADDRESS = '<robot-address.viam.cloud>';
+const String API_KEY = '<api-key>';
+const String API_KEY_ID = '<api-key-id>';
+const String PART_ID = '<part-id>';
+
+final machine = await RobotClient.atAddress(
+    MACHINE_ADDRESS,
+    RobotClientOptions.withApiKey(
+        apiKey: API_KEY,
+        apiKeyId: API_KEY_ID,
+    ),
+);
+
+final client = await ViamClient.withApiKey(
+    apiKeyId: API_KEY_ID,
+    apiKey: API_KEY,
+);
+
+final dataClient = client.dataClient;
+
+final camera = Camera.fromRobot(machine, CAMERA_NAME);
+
+// Capture image
+final imageFrame = await camera.getImage();
+
+// Upload binary data
+final now = DateTime.now().toUtc();
+final fileId = await dataClient.binaryDataCaptureUpload(
+    partId: PART_ID,
+    componentType: 'camera',
+    componentName: CAMERA_NAME,
+    methodName: 'GetImages',
+    dataRequestTimes: [now, now],
+    fileExtension: '.jpg',
+    binaryData: imageFrame,
+);
+
+// Cleanup
+await robotClient.close();
+dataClient.close();
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+Once you've captured [enough images for training](/data-ai/train/train-tflite/), you must [annotate](/data-ai/train/annotate-images/) the images before you can use them to train a model.
+
+## Capture images over time
+
+To capture a large number of images for training an ML model, use the data management service to [capture and sync image data](/data-ai/capture-data/capture-sync/) from your camera.
+
+When you sync with data management, Viam stores the images saved by capture and sync on the [**DATA** page](https://app.viam.com/data/), but does not add the images to a dataset.
+To use your captured images for training, [annotate the images](/data-ai/train/annotate-images/), then [add the images to a dataset](/data-ai/train/update-dataset/).
```

Comment:
```suggestion
To use your captured images for training, [add the images to a dataset](/data-ai/train/update-dataset/) and [annotate them](/data-ai/train/annotate-images/), so you can use them to train a model.
```
To follow order in left nav (I guess it doesn't really matter which order they do it in? Just trying to make it slightly less confusing)